ClassHomework/src/APCSWork/Triangle.java

```diff
@@ -0,0 +1,68 @@
+package APCSWork;
+
+import java.util.ArrayList;
+
+public class Triangle extends Polygon{
+	public Triangle() {
+		super(3, new ArrayList<Point>());
+	}
+	public Triangle(ArrayList<Point> coors) {
+		super(3, coors);
+	}
+
+	public Triangle(Point a, Point b, Point c) {
+		super(3, new ArrayList<Point>());
+		super.addOneCoordinate(a.x, a.y);
+		super.addOneCoordinate(b.x, b.y);
+		super.addOneCoordinate(c.x, c.y);
+		
+	}
+	
+	@Override
+	public double area() {
+		// TODO Auto-generated method stub
+		double base = 0; 
+		return Math.abs((coordinates.get(0).x * 
+				(coordinates.get(1).y - coordinates.get(2).y) 
+				+ coordinates.get(1).x * (coordinates.get(2).y - 
+				coordinates.get(0).y) + coordinates.get(2).x * 
+				(coordinates.get(0).y - coordinates.get(1).y))/2);
```

Comment:
I believe this is a working implementation. But you can clean this up and add readability by creating 3 variables called a, b, c, that calls `coordinates.get(x)`. then you can do something like this which is simpler: 
`return Math.abs(((a.x * (b.y - c.y) + b.x * (c.y - a.y) + c.x * (a.y - b.y))) / 2);`